src/index.html

```diff
@@ -7,6 +7,406 @@
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="product main__product">
+        <div class="product__card" data-qa="card">
+          <div class="product__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac"
+              class="product__image product__image--image"
+            >
+          </div>
+
+          <div class="product__container">
+            <div class="product__desc">
+              <h2 class="product__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+              <p class="product__code">Product code: 195434</p>
+            </div>
+
+            <div class="product__stars">
+              <div class="stars stars--4 product__stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product__reviews">Reviews: 5</p>
+            </div>
+
+            <div class="product__price">
+              <p class="product__price_price">Price:</p>
+              <p class="product__price_value">$2,199</p>
+            </div>
+
+            <a
+              href="#"
+              class="product__button"
+              data-qa="card-hover"
+            >
+              Buy
+            </a>
+          </div>
+        </div>
+
+        <div class="product__card">
+          <div class="product__image">
+            <img
+              src="./images/imac.jpeg"
+              alt="iMac"
+              class="product__image product__image--image"
+            >
+          </div>
+
+          <div class="product__container">
+            <div class="product__desc">
+              <h2 class="product__title">
+                APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+              </h2>
+              <p class="product__code">Product code: 195434</p>
+            </div>
+
+            <div class="product__stars">
+              <div class="stars stars--4 product__stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="product__reviews">Reviews: 5</p>
+            </div>
+
+            <div class="product__price">
+              <p class="product__price_price">Price:</p>
+              <p class="product__price_value">$2,199</p>
+            </div>
+
+            <a
+              href="#"
+              class="product__button"
+              data-qa="hover"
```

Comment:
You should remove all old data-qa and leave only data-qa="nav-hover", data-qa="card" and data-qa="card-hover". Read 9 point in requirements